Makefile

```diff
@@ -19,6 +19,7 @@ clean:
 	rm -f *.gcda *.gcno generate.info
 	rm -f tests/ctests/*.gcda tests/ctests/*.gcno
 	rm -rf test-coverage .coverage coverage.xml
```

Comment:
nitpick: coverage.xml would be best suited here

Reply:
Moved into #338 338